code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -74,6 +74,51 @@
 /obj/item/weapon/cautery/is_hot()
 	return 1
 
+/obj/item/weapon/cautery/laser
+	sharpness = 1.5
+	sharpness_flags = SHARP_TIP | SHARP_BLADE | HOT_EDGE
+	damtype = "fire"
+	force = 10.0
+	throwforce = 5.0
+	throw_speed = 3
+	throw_range = 5
+	attack_verb = list("attacks", "slashes", "stabs", "slices", "tears", "rips", "dices", "cuts")
```

Comment:
I'm not sure it makes sense for a laser to "stab," "tear," "rip," et cetera. A laser that functions as a cautery should arguably not have any verbs that deal with sharpness.

Reply:
A small oversight on my part, I will make it more cautery like for the cautery mode.